test/unit/models/events.test.js

```diff
@@ -74,4 +76,117 @@ describe("Events", function () {
       }
     });
   });
+
+  describe("addPeerToEvent", function () {
+    it("should create a new peer document if it doesn't exist", async function () {
```

Comment:
NIT 
```suggestion
    it("Should create a new peer document if it doesn't exist", async function () {
```

Reply:
"S" is small everywhere. So, following the same for consistency.